### PR TITLE
DDF-1275: reorganized DDF Content documentation.

### PR DESCRIPTION
**Pull request by ricklarsen**
@stustison @clockard @oscaritoro @ahoffer @vinamartin @djblue @gordocanchola

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/452" target="_blank">ReviewNinja

**Comment by clockard** [comment]
LGTM  :+1:

File: catalog/content/docs/src/main/resources/IntContents.adoc [hunk 1]
```diff
@@ -413,69 +446,83 @@ The following storage providers are provided with the Content Framework out of
...
+The `inputTransformer` service accepts the content item's `InputStream` and parses it, creating a Metacard that is returned to the Content Cataloger Plugin.
+This Metacard is then used in the `create` and `update` operations invoked on the Catalog Framework to interface with the Metadata Catalog.
 
-==== Using
+===== Using
 
-Use the Content Cataloger Plugin if create/update/delete of catalog entries in the Metadata Catalog based on the content item are desired. These CUD operations on the Metadata Catalog are made possible by parsing the content item to create a metacard and then using this metacard in the CUD operations on the Catalog Framework. The Content Cataloger Plugin is the only component in the DDF Content Framework that has the ability to interface with the Catalog Framework (and hence the Metadata Catalog).
+Use the Content Cataloger Plugin if create/update/delete of catalog entries in the Metadata Catalog based on the content item are desired.
+These CUD operations on the Metadata Catalog are made possible by parsing the content item to create a metacard and then using this metacard in the CUD operations on the Catalog Framework.
```
Comment (vinamartin) [inline catalog/content/docs/src/main/resources/IntContents.adoc]: CRUD?

**Comment by vinamartin** [comment]
LGTM :+1:

File: distribution/docs/src/main/resources/ManContents.adoc [hunk 2]
```diff
@@ -1853,6 +1853,7 @@ The variable version is longer but will not need to be changed if the system hos
 A banner is displayed on the admin console notifying "The system is insecure because default configuration values are in use."
 
 A detailed view is available of the properties to update.
+====
```
Comment (oscaritoro) [inline distribution/docs/src/main/resources/ManContents.adoc]: This line is duplicating the intent of line 1858 and breaking the generation of the docs from this point forward :(

File: catalog/content/docs/src/main/resources/IntContents.adoc (hunk 1 above)
Comment (ricklarsen) [inline catalog/content/docs/src/main/resources/IntContents.adoc]: I wondered what happened to "retrieve." Fixed.

File: distribution/docs/src/main/resources/ManContents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/ManContents.adoc]: Looks like a bad merge/rebase.

File: catalog/content/docs/src/main/resources/ExtContents.adoc [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
-
-* ddf.content
-* ddf.content.data
-* ddf.content.operation
-* ddf.content.plugin
-* ddf.content.storage
-* ddf.content.util
-* ddf.content.core.directorymonitor
\ No newline at end of file
+`ddf.camel.component.content`
```
Comment (oscaritoro) [inline catalog/content/docs/src/main/resources/ExtContents.adoc]: Just checking... is it truly your intent to have these all render inline with just one blank space separating each one? Are all package lists moving from bullet lists over to this style?

Comment (oscaritoro) [inline catalog/content/docs/src/main/resources/ExtContents.adoc]: LGTM after you address my package list question :+1:

Comment (ricklarsen) [inline catalog/content/docs/src/main/resources/ExtContents.adoc]: fixed.

**Comment by oscaritoro** [comment]
LGTM :+1:

**Pull request merged**